java gr 8/homework/07homework7_obj_containers/src/main/java/ev/purchase/program/domain/PurchaseEvCar.java

```diff
@@ -0,0 +1,36 @@
+package ev.purchase.program.domain;
+
+/**
+ * Created by Narci on 9/24/2017
+ */
+public class PurchaseEvCar {
```

Comment:
Why don't you call this class Order? Is it defined by the requirements? Do we need it?